packages/client/src/rpc/modules/debug.ts

```diff
@@ -278,4 +295,70 @@ export class Debug {
       message: err.message.toString(),
     }
   }
+
+  /**
+   * Returns a limited set of storage keys belonging to an account.
+   * @param params An array of 5 parameters:
+   *    1. The hash of the block at which to get storage from the state.
+   *    2. The transaction index of the requested block post which to get the storage.
+   *    3. The address of the account.
+   *    4. The starting (hashed) key from which storage will be returned. To include the entire range, pass '0x00'.
+   *    5. The maximum number of storage values that could be returned.
+   * @returns A {@link StorageRange} object that will contain at most `limit` entries in its `storage` field.
+   * The object will also contain `nextKey`, the next (hashed) storage key after the range included in `storage`.
+   */
+  async storageRangeAt(params: [string, number, string, string, number]) {
+    const [blockHash, txIndex, account, startKey, limit] = params
+
+    if (this.vm === undefined) {
+      throw new Error('Missing VM.')
+    }
+
+    if (txIndex < 0) {
```

Comment:
if we don't already have a validator for `index` parameters, it could be worth adding one. especially if there are other methods are doing this themselves